downstream/modules/platform/proc-defining-node-types.adoc

```diff
@@ -5,7 +5,7 @@
 = Defining {AutomationMesh} node types
 
 [role="_abstract"]
-You can a define node type either by its default value assigned by the inventory group or by using the `node_type` host variable. Specify the `node_type` either as part of the inventory group or within the inventory `vars` group.  This section provides examples of how you can define node types in your inventory file.  Nodes in `[execution_nodes]` default execution node_type. Hybrid node types can be overridden to be control type via `node_type=control`. Execution node type can be overridden to be hope node type via `node_type=hop`
+You can a define node type either by its default value assigned by the inventory group or by using the `node_type` host variable. Specify the `node_type` either as part of the inventory group or within the inventory `vars` group.  This section provides examples of how you can define node types in your inventory file.  Nodes in `[execution_nodes]` default execution node_type. Hybrid node types can be overridden to be control type via `node_type=control`. Execution node type can be overridden to be hop node type via `node_type=hop`
```

Comment:
Instead of via `note_type=control', say 'using the `node_type=control` variable.
Also type via 'node_type=hop', say 'type using `node_type=hop'

I've documented node_type as an inventory file variable and it would be node_type = 'hop'. Is that worth checking?

Reply:
@ianf77 I just corrected a typo in this paragraph. I think it needs restructuring.